star_align.py

```diff
@@ -70,6 +70,48 @@ def scan_workdir(base):
     raise Exception("Unable to determine input type")
 
 
+def spreadsheet2dict(spreadFile):
```

Comment:
This function is implemented in the Python standard library as cvs.DictReader ( https://docs.python.org/2/library/csv.html#csv.DictReader )